src/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+
+const operation = process.argv[2];
+
+if (operation !== 'add' && operation !== 'multiply') {
+  // eslint-disable-next-line no-console
+  console.log('Operation is not supported!');
+
+  return;
+}
+
+const params = process.argv.slice(3);
+const numericParams = params.filter((a) => (
+  typeof parseInt(a) === 'number' && !isNaN(parseInt(a))
+));
+
+if (!numericParams.length) {
+  // eslint-disable-next-line no-console
+  console.log(`There is nothing to ${operation}`);
+
+  return;
+}
+
+if (operation === 'add') {
+  // eslint-disable-next-line no-console
+  console.log(numericParams.reduce((a, b) => parseInt(a) + parseInt(b)));
+} else if (operation === 'multiply') {
```

Comment:
You can switch the operation to make your code cleaner. 
Btw, the default case can print the error, so you will not need the first check

Reply:
My logic of putting the first check in the beginning was simply to stop executing it straight away and not waste time to continue with other actions if the operation is not supported.
But if it's better with switch, ok )